src/components/aria-live/README.md

```diff
@@ -0,0 +1,37 @@
+# MdAriaLive
```

Comment:
I kind of want to call this `md-live-announcer`. 

@hansl @kara @robertmesserle thoughts?